cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/openai/OpenAIEmbedding.scala

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.cognitive.openai
+
+import com.microsoft.azure.synapse.ml.cognitive.{CognitiveServicesBase, HasCognitiveServiceInput, HasServiceParams}
+import com.microsoft.azure.synapse.ml.core.contracts.HasInputCol
+import com.microsoft.azure.synapse.ml.io.http.JSONOutputParser
+import com.microsoft.azure.synapse.ml.logging.SynapseMLLogging
+import com.microsoft.azure.synapse.ml.param.ServiceParam
+import org.apache.http.entity.{AbstractHttpEntity, ContentType, StringEntity}
+import org.apache.spark.ml.ComplexParamsReadable
+import org.apache.spark.ml.linalg.SQLDataTypes.VectorType
+import org.apache.spark.ml.util._
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.types._
+import spray.json.DefaultJsonProtocol._
+import spray.json._
+import org.apache.spark.ml.linalg.{Vector, Vectors}
+
+import scala.language.existentials
+
+object OpenAIEmbedding extends ComplexParamsReadable[OpenAIEmbedding]
+
+class OpenAIEmbedding (override val uid: String) extends CognitiveServicesBase(uid)
+  with HasServiceParams with HasInputCol with HasAPIVersion with HasDeploymentName
```

Comment:
I dont think there needs to be an InputCol here or elsewhere. Use the promptCol and other setters